server/src/main/java/com/testsigma/service/TestsigmaLabDriverSettingsService.java

```diff
@@ -26,6 +29,9 @@ public class TestsigmaLabDriverSettingsService extends DriverSettingsService {
   public static final String PLATFORM_MOBILE_URL = "%s://%s:%s@%s/mobile/wd/hub";
   public static final String PLATFORM_MOBILE_URL_WITH_PORT = "%s://%s:%s@%s:%s/mobile/wd/hub";
 
+  protected TestDeviceEntity testDeviceEntity;
+  protected com.testsigma.automator.entity.TestDeviceSettings settings;
```

Comment:
@vikramvr20 server tru to avoid automator classes import

Reply:
Addressed